### ISSUE TITLE
Video of the robot operation in the room

### ISSUE THREAD

**Issue by mxlfrbt** [PacktPublishing/Artificial-Intelligence-for-Robotics-2e#4]
Hi,
Do you have video of the robot operation? I can't find it.

Thanks

**Comment by FGovers** [comment]
I don't have any videos posted online.  I'll put some together - it's going to take me a bit. We plan to do a video series to accompany the book that will be available shortly

**Comment by mxlfrbt** [comment]
> I don't have any videos posted online. I'll put some together - it's going to take me a bit. We plan to do a video series to accompany the book that will be available shortly

Yes, I can understand, video making process is a really tough process. But if you will have some simple one-shot no edit video in the end of the each chapter showing how robot behaves, will be great.
